.travis.yml

```diff
@@ -1,36 +0,0 @@
-language: php
```

Comment:
Can you please add Symfony 6 to the versions here please? https://github.com/mybuilder/cronos/blob/master/.github/workflows/test.yml#L12
We're now using github actions for testing rather than github

Reply:
I have deleted the whole file. As it is not incluced anymore in the `master` branch.

Reply:
Can you update your branch and add Symfony 6 please to the linked github action?

Reply:
The github actions needs more changes. Only adding 6.0.0 to the matrix will fail.
https://github.com/Chris53897/cronos/actions/runs/3463286751/jobs/5783304065

Reply:
Ah of course Symfony 6 requires PHP8. 
If you have time to update the action then that would be good. If not, then take it out for now and I'll update it when I get a chance

Reply:
I will try to find some time after my vacation. Should not be so hard, i guess.
Something like this.
`matrix:
     include:` 
https://github.com/Chris53897/KnpMenuBundle/blob/master/.github/workflows/build.yaml

Reply:
Thank you, enjoy your holiday 😎 
Something like that matrix would work yes 👍

Reply:
Thanks. I have adjusted the ci-tests.
And i changed composer.json to "^3.4|^4.4|^5.4|^6.0", as supported LTS.
Additional i fixed a deprecation message of the ci.

If you want more or other test runs, please let me know or change it to your needs.